changelogs/unreleased/1261-asaf-erlich

```diff
@@ -0,0 +1 @@
+Update ark restore to not open every single file open during extraction of the datark restore to not open every single file open during extraction of the data
```

Comment:
Looks like your message got duplicated. Can you also replace `ark` with `velero`? Thanks :)